weave/decorator_type.py

```diff
@@ -44,6 +45,10 @@ def wrap(target):
             )
         else:
             bases = (base_type,)
+
+        if __mixins is not None:
```

Comment:
I searched for this usage and didn't see it anywhere - maybe dev tracks?

Reply:
This is used here

https://github.com/wandb/weave/pull/177/files#diff-abdb20bee21d5a10c8054b1964ad13c5e9b01e9d128686bc3d67e5b9e6e80092R69